src/infra/nodemailer/setup.js

```diff
@@ -0,0 +1,41 @@
+import nodemailer from 'nodemailer';
+import * as dotenv from 'dotenv';
+import { generatingQRCode } from '../../utils/generateQRCode.js';
+
+dotenv.config();
+
+let transporter = nodemailer.createTransport({
+  host: process.env.email_host,
+  port: process.env.email_port,
+  secure: false,
+  service: process.env.email_service,
+  auth: {
+    user: process.env.email_user,
+    pass: process.env.email_pass
+  }
+});
+
+export async function sendEmail(studentData) {
+
+  const QRCode = await generatingQRCode();
+
+  var message = {
+    from: process.env.email_user, 
+    to: studentData.email,
+    subject: 'QRCode de Acesso',
+    html: `<p>Este é o seu QRCode de acesso:</p><img src="cid:qr-code-cid" alt="QR code">`,
+    attachments: [{
+      filename: 'qrcode.png',
+      path: `${QRCode}`,
+      cid: 'qr-code-cid'
```

Comment:
Seguindo a [documentação](https://nodemailer.com/message/embedded-images/) do nodemailer, seria interessante colocar o CID em um formato padronizado.

```suggestion
    html: `<p>Este é o seu QRCode de acesso:</p><img src="cid:student_qr_code@ifpeopensource.com.br" alt="QR code">`,
    attachments: [{
      filename: 'qrcode.png',
      path: `${QRCode}`,
      cid: 'student_qr_code@ifpeopensource.com.br'
```

Provavelmente seria melhor colocar o host do e-mail ao invés de `@ifpeopensource.com.br` hardcoded.

Reply:
Certo.